controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -33,6 +33,11 @@ def record_video(supervisor: Supervisor, file_path: Path) -> Iterator[None]:
     print("Saving video to {}".format(file_path))
 
     config = controller_utils.get_recording_config()
+    if config.quality == 0:
+        print('Not recording movie')
+        yield
```

Comment:
Is the `yield` needed here? Can we just `return`?

Reply:
The `yield` is required otherwise this context manager exits immediately and the match is never run